### DISCUSSION TITLE
Is it possible to use body parameters in remote_json config?

### DISCUSSION THREAD

**Discussion by george-weaverhq** [ory/oathkeeper#845] [Q&A] [answered]
I would like to pass a request through oathkeeper.

I would like to first authenticate using ory kratos, and then use the remote_json authorizer, in order to check for permissions using ory keto.

Is it possible to use elements of the original request's body in the remote_json handler config, in order to make the authorization calling the keto/check endpoint?

---
remote_json:
   enabled: true
    config:
      remote: http://keto:4466/check
      payload: |
        {
          "subject": "x",
          "relation": "write",
          "object": "projectName",
          "namespace": "projects"
        }
---

So in the above example, I would like the payload to be taken from the body of the original request and not use url matching.

**Comment by vinckr** [comment, marked as answer]
I think should be possible, see this for example:
https://github.com/radekg/ory-reference-compose/blob/master/compose/configs/oathkeeper/rules.json

Would you be willing to share a rough sketch of your setup?

**Reply by denysandriyanov** [reply]
Could you please explain the example you mentioned?

"authorizer": {
            "handler": "remote_json",
            "config": {
                "remote": "http://keto:4466/check",
                "payload": "{\"namespace\": \"default-namespace\", \"subject\": \"{{ print .Subject }}\", \"object\": \"token:and:keto\", \"relation\": \"get\"}"
            }
        },

where are you actually getting some values here from original request body?

**Comment by denysandriyanov** [comment]
@george-weaverhq @vinckr Could you please explain if and how you managed to get values from the original request body and pass it to the authorizer?
